### ISSUE TITLE
Error in Step 4 

### ISSUE THREAD

**Issue by Atalasia** [sigven/pcgr#84]
```
...
2019-08-26 10:08:48 - pcgr-writer - INFO - STEP 4: Generation of output files - variant interpretation report for precision oncology
2019-08-26 10:09:13 [INFO] Initializing PCGR report - sample CosmicCodingMuts.chr1
2019-08-26 10:09:30 [INFO] Excluding 0 variants from non-nuclear chromosomes/scaffolds
2019-08-26 10:09:33 [INFO] Number of PASS variants: 61911
2019-08-26 10:09:33 [INFO] Number of SNVs: 59290
2019-08-26 10:09:33 [INFO] Number of deletions: 1843
2019-08-26 10:09:33 [INFO] Number of insertions: 659
2019-08-26 10:09:33 [INFO] Number of block substitutions: 111
2019-08-26 10:09:33 [INFO] Extending annotation descriptions related to UniprotKB/SwissProt protein features
2019-08-26 10:09:36 [INFO] Extending annotation descriptions related to Database of Curated Mutations (DoCM)
2019-08-26 10:09:38 [INFO] Extending annotation descriptions related to KEGG pathways
2019-08-26 10:09:38 [INFO] Extending annotation descriptions related to ClinVar
2019-08-26 10:10:02 [INFO] ------
2019-08-26 10:10:02 [INFO] Generating data for tiered cancer genome report - somatic calls tier model 'pcgr_acmg'
2019-08-26 10:10:02 [INFO] Number of protein-coding variants: 45225
2019-08-26 10:10:02 [INFO] Looking up SNV/InDel biomarkers for precision oncology - any tumortype
Error in rbind(deparse.level, ...) :
  numbers of columns of arguments do not match
Calls: <Anonymous> -> <Anonymous> -> <Anonymous> -> rbind -> rbind
Execution halted
```

Hello, 
I am running COSMIC VCF split into chromosomes (all left aligned and repeats removed) and some chromosomes (about 2/3 of them) are giving me the above error and some finish without any errors. 

I am using the most recent version of PCGR.

**Comment by sigven** [comment]
Hi @Atalasia,
Thanks for reporting! I have figured out the bug and will make a push with a fix next week. As another note, please be aware that large input sets may cause some trouble for the rendering in the HTML report (PCGR is intended for somatic callsets in individual tumors, which are usually < 30,000 variants)

best,
Sigve

**Comment by Atalasia** [comment]
Thanks for the reply. 

I'll look forward to it. 
I'll split the file into smaller pieces if variant count goes above 30,000.

**Comment by sigven** [comment]
Hi again,

As a test, I downloaded the COSMIC coding mutations VCF file (v89). I noticed that a number of the exact same mutations appear multiple times (yet with the different COSMIC mutation IDs), these were merged when making a COSMIC "non-redundant" VCF. Since the error you encountered occurred during biomarker lookup, I intersected the VCF file with all genes that are actionable (i.e. with actionable variants according to CIViC++). I then split the final VCF file (~160,000 variants or so) into smaller chunks (20,000 variants), and ran these through PCGR (latest, unreleased version). One chunk (number 5) got an error during  HTML generation, and I suspect this is a memory issue within the Docker container, I need to revisit that. All others got through OK.

Results available [here](https://www.dropbox.com/sh/x48tlo6tl3cqzmk/AABWuHU76zRt-_Kxw8g2g5b9a?dl=0), fixes will be pushed shortly.

Thanks again for bug reporting!

Sigve
